app/src/main/java/org/groundplatform/android/persistence/remote/firebase/schema/LoiCollectionReference.kt

```diff
@@ -59,6 +59,13 @@ class LoiCollectionReference internal constructor(ref: CollectionReference) :
         .whereEqualTo(OWNER_FIELD, ownerUserId),
     )
 
+  /** Retrieves LOIs created by the anyone in the specified survey. Main-safe. */
```

Comment:
Why "created by.."? Also, what are "public" LOIs? we only have to types of LOIs, predefined and ad hoc. If a survey allows data collectors to view each others LOI's, we just need to query LOIs with the `predefined !== false`  filter.

Reply:
Fixed